docs/orchestrating-elastic-stack-applications/elasticsearch/snapshots.asciidoc

```diff
@@ -285,6 +285,115 @@ PUT /_snapshot/my_s3_repository
+      --encryption-services blob
+----
++
+<1> This can be any of the supported storage account types `Standard_LRS`, `Standard_ZRS`, `Standard_GRS`, `Standard_RAGRS` but not `Premium_LRS` see https://www.elastic.co/guide/en/elasticsearch/reference/current/repository-azure.html[the Elasticsearch documentation] for details.
++
+. Create a container in the storage account, for this example `es-snapshots`.
++
+[source,sh]
+----
+az storage container create \
+   --account-name $STORAGE_ACCOUNT_NAME \
+   --name es-snapshots --auth-mode login
+----
++
+. Create a role assignment between the managed identity and the storage account.
++
+[source,sh,subs="callouts"]
+----
+STORAGE_SCOPE=$(az storage account show \
+  --resource-group $RESOURCE_GROUP \
+  --name $STORAGE_ACCOUNT_NAME --query id -o tsv | sed 's#/##') <1>
+
+az role assignment create \
+  --assignee-object-id $USER_ASSIGNED_IDENTITY_OBJECT_ID \
+  --role "Storage Blob Data Contributor" \
+  --scope $STORAGE_SCOPE
+----
++
+<1> The storage account ID needs to be specified as the scope for the role assignment without the leading slash returned by the `az storage account show` command.
++
+. Create a Kubernetes secret, called `keystore` in this example, with the storage account name. This is necessary to be able to specify the account name as a secure setting in Elasticsearch in the next step.
++
+[source,sh]
+----
+kubectl create secret generic keystore \
+  --from-literal=azure.client.default.account=${STORAGE_ACCOUNT_NAME}
+----
++
+. Create an Elasticsearch cluster that uses the Kubernetes service account created earlier.
++
+[source,yaml,subs="attributes,callouts"]
+----
+apiVersion: elasticsearch.k8s.elastic.co/v1
+kind: Elasticsearch
+metadata:
+  name: az-workload-identity-sample
+spec:
+  version: {version} 
+  secureSettings:
+  - secretName: keystore
+  nodeSets:
+  - name: default
+    count: 1
+    podTemplate:
+      metadata:
+        labels:
+          azure.workload.identity/use: "true" 
+      spec:
+        serviceAccountName: workload-identity-sa <1>
+        containers:
+        - name: elasticsearch
+          env:
+          - name: AZURE_FEDERATED_TOKEN_FILE <2>
+            value: /usr/share/elasticsearch/config/azure/tokens/azure-identity-token
+          volumeMounts:
+          - name: azure-identity-token
+            mountPath: /usr/share/elasticsearch/config/azure/tokens <2>
+----
++
+<1> This is the service account created earlier in the steps from the  https://learn.microsoft.com/en-us/azure/aks/workload-identity-deploy-cluste[Azure Workload Identity] tutorial.
+<2> The corresponding volume is injected by the https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html[Azure Workload Identity Mutating Admission Webhook]. For Elasticsearch to be able to access the token, the mount needs to be in a sub-directory of the Elasticsearch config directory. The corresponding environment variable needs to be adjusted as well.
```

Comment:
Nit: 
- Add a <1> on the secret:
```
  - secretName: keystore <1>

---
<1> This is the secret created earlier in the steps with the storage account name.
```

- Repeat <2> for the label:
```
    azure.workload.identity/use: "true" <2>
```
- hence, rename your <1> in <3>, because of the new <1>.